### DISCUSSION TITLE
Mouse button as keybind?

### DISCUSSION THREAD

**Discussion by ZeroZen451** [SaifAqqad/AHK_MicMute#111] [Q&A]
Is it possible to use your mouse buttons as your keybind for muting/unmuting?

**Comment by SaifAqqad** [comment]
only mouse back and forward buttons

**Reply by Lexonixeo** [reply]
how to record this buttons? i'm pressing Forward mouse button and... "Recording"

**Reply by SaifAqqad** [reply]
> how to record this buttons? i'm pressing Forward mouse button and... "Recording"

it should display XButton2 when you press the mouse forward button
<img width="424" height="99" alt="image" src="https://github.com/user-attachments/assets/028b2d6b-6775-4ab9-99c6-91958851b307" />


maybe your mouse back/forward buttons are remapped to something else in the mouse's software? (i mean like razer synapse or hyperx ngenuity)
